modules/apps/redirect/redirect-web/src/main/resources/META-INF/resources/view_redirect_entries.jsp

```diff
@@ -78,31 +78,31 @@ RedirectManagementToolbarDisplayContext redirectManagementToolbarDisplayContext
 						%>
 
 						<liferay-ui:search-container-column-text
-							cssClass="table-cell-expand"
+							cssClass="table-cell-expand table-cell-text-truncate-reverse"
 							name="source-url"
 						>
 
 							<%
 							String sourceURL = HtmlUtil.escape(RedirectUtil.getGroupBaseURL(themeDisplay) + StringPool.SLASH + redirectEntry.getSourceURL());
 							%>
 
-							<span class="text-truncate text-truncate-reverse" data-title="<%= HtmlUtil.escapeAttribute(sourceURL) %>">
-								<bdi><%= sourceURL %></bdi>
-							</span>
+							<bdi data-title="<%= HtmlUtil.escapeAttribute(sourceURL) %>">
```

Comment:
<bdi> has no support for some browsers https://developer.mozilla.org/es/docs/Web/HTML/Element/bdi

Reply:
Yep, in the worst case, it will work as before. Previously discussed https://github.com/liferay-frontend/liferay-portal/pull/940 and in FI thread.